packages/js/plugins/ethereum/src/__tests__/integration/polywrap.build.yaml

```diff
@@ -1,6 +1,9 @@
-format: 0.1.0
-config:
-  node_version: "16.13.0"
+format: 0.2.0
+strategies:
+  image:
+    node_version: "14.16.0"
```

Comment:
nit: would it be possible to use `16.13.0` in all the custom build manifests (for as) in monorepo?

Reply:
I think this should probably be done in a separate PR